src/main/java/io/r2dbc/postgresql/PostgresqlConnectionConfiguration.java

```diff
@@ -127,6 +135,14 @@ boolean isForceBinary() {
         return this.forceBinary;
     }
 
+    boolean getSsl() {
```

Comment:
We should return a value object here that encapsulates SSL usage. See comment below on `ReactorNettyClient.connect(…)`.